scriptworker/cot/verify.py

```diff
@@ -1277,6 +1278,7 @@ def _get_action_perm(action_defn):
             'taskId': parent_link.task_id,
             'now': jsone_context['now'],
             'as_slugid': jsone_context['as_slugid'],
+            'taskId': json_context['as_slugid']('decision'),
```

Comment:
This should probably be `jsone_context`

Reply:
I don't think this wants to change, as this is the action task json-e context being generated.